app/uk/gov/hmrc/fhregistrationfrontend/views/registrationstatus/StatusPageParams.scala

```diff
@@ -34,65 +35,70 @@ case class StatusPageParams(
-  val statusParams = List(
-    StatusPageParams(
-      Received,
-      cta = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.AmendmentController.startAmendment(),
-      secondary = Some(uk.gov.hmrc.fhregistrationfrontend.controllers.routes.WithdrawalController.startWithdraw()),
-      nextTemplate = statusWhatHappensNext.apply,
-      showSubHeading = false
-    ),
-    StatusPageParams(
-      Processing,
-      cta = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.AmendmentController.startAmendment(),
-      secondary = Some(uk.gov.hmrc.fhregistrationfrontend.controllers.routes.WithdrawalController.startWithdraw()),
-      nextTemplate = statusWhatHappensNext.apply,
-      showSubHeading = false
-    ),
-    StatusPageParams(
-      Approved,
-      cta = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.AmendmentController.startVariation(),
-      secondary = Some(uk.gov.hmrc.fhregistrationfrontend.controllers.routes.DeregistrationController.startDeregister()),
-      nextTemplate = statusWhatHappensNext.apply,
-      category = "post"
-    ),
-    StatusPageParams(
-      ApprovedWithConditions,
-      cta = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.AmendmentController.startVariation(),
-      secondary = Some(uk.gov.hmrc.fhregistrationfrontend.controllers.routes.DeregistrationController.startDeregister()),
-      nextTemplate = statusWhatHappensNext.apply,
-      category = "post"
-    ),
-    StatusPageParams(
-      Rejected,
-      cta = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.startOrContinueApplication(),
-      secondary = Some(uk.gov.hmrc.fhregistrationfrontend.controllers.routes.ReadOnlySummaryController.view()),
-      nextTemplate = statusNewApplication.apply,
-      showSubHeading = false,
-      category = "post"
-    ),
-    StatusPageParams(
-      Revoked,
-      cta = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.startOrContinueApplication(),
-      secondary = Some(uk.gov.hmrc.fhregistrationfrontend.controllers.routes.ReadOnlySummaryController.view()),
-      nextTemplate = statusNewApplication.apply,
-      category = "post"
-    ),
-    StatusPageParams(
-      Withdrawn,
-      cta = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.startOrContinueApplication(),
-      secondary = None,
-      nextTemplate = statusWithdrawn.apply,
-      showSubHeading = false
-    ),
-    StatusPageParams(
-      Deregistered,
-      cta = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.startOrContinueApplication(),
-      secondary = Some(uk.gov.hmrc.fhregistrationfrontend.controllers.routes.ReadOnlySummaryController.view()),
-      nextTemplate = statusNewApplication.apply,
-      category = "no-notification"
-    )
-  )
 
-  def apply(status: FhddsStatus): Option[StatusPageParams] = statusParams find (_.status == status)
+  class StatusParams @Inject()(registrationStatusViews: RegistrationStatusViews) {
```

Comment:
I think we could remove the object and just keep the class instead of having the class inside the object? Check where this is used if you can inject class directly.